tests/test_text_generation_example.py

```diff
@@ -71,6 +71,9 @@
         "torch_compile_distributed": [
             ("meta-llama/Llama-2-7b-hf", 39.72973199515235),
         ],
+        "distributed_tp": [
+            ("meta-llama/Llama-2-7b-hf", 1856.8140409694543),
```

Comment:
I get 1345.2369318328463 on my Gaudi2 instance with 1.16. Could you check if you get the same? I'm assuming the gap comes from 1.17, but I'm really not sure about it.

Reply:
@regiss I observed a performance number of 1361.2050636537415 on the 1.16 release, which is close what you have observed. I have updated the expected performance number accordingly.